app/apptesting/test_helpers.go

```diff
@@ -280,3 +281,12 @@ func  (s *AppTestHelper) ICAPacketAcknowledgement(msgs []sdk.Msg) channeltypes.A
 	ack := channeltypes.NewResultAcknowledgement(marshalledTxMsgData)
 	return ack
 }
+
+// Get an IBC denom from it's native host denom
+// This assumes the transfer channel is channel-0
+func (s *AppTestHelper) GetIBCDenom(denom string) transfertypes.DenomTrace {
+	sourcePrefix := transfertypes.GetDenomPrefix(ibctesting.TransferPort, ibctesting.FirstChannelID)
```

Comment:
This function is really just doing string formatting (i.e. port/channelId/denom) and storing it as a trace type. It's not checking anything from state.

That said, maybe I could name this function a little more clearly? Fwiw, I originally had this return the actual IBC hash, but then changed it to return just the denom object since that's what you need to store it in state, and you can easy get the hash with `.IBCDenom()`